pages/index.tsx

```diff
@@ -247,6 +251,7 @@ export async function getServerSideProps(context: any) {
       donationStep = 0;
     }
   }
+  const resolvedUrl = context.resolvedUrl;
```

Comment:
Is it correct that http://localhost:3000/?to=proj_6x3GTD5cMRv0OeQAiIlJZ0Au results into e.g. `<meta property="twitter:image" content="https://s.wordpress.com/mshots/v1/http%3A%2F%2Flocalhost%3A3000%2F%3Fto%3Dproj_6x3GTD5cMRv0OeQAiIlJZ0Au?w=1200&amp;h=770.jpg">`

Reply:
https://s.wordpress.com/mshots/v1/https%3A%2F%2Fdonate-with-planet-m434g86p1-planetapp.vercel.app%2F%3Fto%3Dproj_6x3GTD5cMRv0OeQAiIlJZ0Au?w=1200&amp;h=770.jpg

I checked the image in the preview link of this PR it works. Or maybe I didn't understand your question.